Oliver-ke/Teamwork
# #28947267 Implement users create gif feature

#### What does this PR do?
- allows employee to create and share gif

#### Description of Task to be completed?
- add necessary endpoint
- add Cloudinary configuration
- add Cloudinary package

#### How should this be manually tested?
- clone this repository
- run `npm install` to install all related dependencies
- setup .env file to include cloudinary secret config
- run `npm run test` to run test
- you should see a passing tests
- run the command `npm run start:dev` to stat the server
- open postman and make a post request to `localhost:5000/api/v1/gifs` with valid data

#### Any background context you want to provide?
- none

#### What are the relevant project board stories?
[#28947267](https://github.com/Oliver-ke/Teamwork/projects/1#card-28947267)

#### Screenshots (if appropriate)
- none

#### Questions:
